### ISSUE TITLE
[Idea] reduce momentum lost/acquired by build/destroy buildings

### ISSUE THREAD

**Issue by bmorel** [Unvanquished/gameplay#41]
Related Problem issues: #28 

Divide the momentum earned/lost by buildings by 2.

**Disclaimer**
>Ideas without a related problem issue and lacking explanation or depth are closed after a certain time.

**Comment by bmorel** [comment]
The problem this wants to help with for 0.53 is to punish less builders.
Currently, building in punished in 2 ways:

1. it makes you team loose BPs, eventually for long when game growths older. This is rather fine.
2. it makes enemy team earn lot of BPs when they just keep some defenses and other ones roam to hunt buildings. This is not fine at all, as building shuold imo be encouraged.

I believe reducing the momentum loss/earn with buildings would help a bit with the punishment, and is quite easy to implement. It could be done for 0.53.

**Comment by DolceTriade** [comment]
How do we balance this with camping? The whole purpose of momentum for building was to encourage killing bases rather than killing enemies.

**Comment by bmorel** [comment]
The current mechanism in practice forces people to camp, because it's impossible to build outposts, those are immediately punish by BP loss + feeding enemy's momentum.
So, if the intent of the double punishment is to prevent camping, it's a huge failure imo.

**Comment by DolceTriade** [comment]
yes, the current system is definitely inadequate in solving the issue of encouraging players to leave their base. I'm not convinced rewards for building are going to help that significantly.

**Comment by bmorel** [comment]
Current system already rewards building, but it's so dangerous it's not worth the risk at all.
It pays much more to build a bunker (for humans, at least, because *them*, they can) and camp it. Any outpost that would be used to control the map in early games lead to enemy team taking momentum advantage, plus the BP debt if the game is not *that* early.
Most of the games I've seen where a player tries to build map control resulted in failure for their team.

I'm not talking about increasing the reward, I'm talking about diminishing it, for both the builder and the destroyer.
Building should weight less in momentum, because there is already the BP debt which can punish strongly enough, when you no longer can rebuild critical stuff.

**Comment by Gireen** [comment]
I don't think this has any positive effect on #28 
It gives even more reason to build a small base since the earned momentum is lesser and killing enemies becomes a more attractive source for it.

**Comment by Viech** [comment]
Have I retitled this adequately? I'm not sure if the issue is about *both* building and destroying as the old title suggested. [I've removed the "lost" part as I found it misleading: momentum is only lost by deconstruction (tied to the construction reward), which is discussed in #55.]

**Comment by mjstahlberg** [comment]
I think that both building (which at some point requires expanding the base) and destroying stuff are good actions that we want to reward in general. I would not be opposed to subtly moving the values for building and destroying closer together (to help with #28) but lowering both significantly I'm not convinced of.

Worth mentioning, when you have an enemy that camps really hard and you are lacking the anti-base weapons to break it, expanding your base may be the best or only option to acquire the needed momentum.
